<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseBtn">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadBtn">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fitToWindowBtn">
        <property name="text">
         <string>Fit to window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetZoomBtn">
        <property name="text">
         <string>Reset zoom</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="GraphicsViewEx" name="graphicsView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsViewEx</class>
   <extends>QGraphicsView</extends>
   <header>graphicsviewex.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
